lstchain/visualization/plot_calib.py

```diff
@@ -0,0 +1,339 @@
+
+from matplotlib import pyplot as plt
+from ctapipe.visualization import CameraDisplay
+import numpy as np
+from matplotlib.backends.backend_pdf import PdfPages
+from ctapipe.instrument import  CameraGeometry
+
+# read back the monitoring containers written with the tool calc_camera_calibration.py
+from ctapipe.io.containers import FlatFieldContainer, WaveformCalibrationContainer, PedestalContainer, \
+    PixelStatusContainer
+
+from ctapipe.io.hdf5tableio import HDF5TableReader
+
+__all__ = ['read_file',
+           'plot_all'
+           ]
+
+ff_data = FlatFieldContainer()
+ped_data = PedestalContainer()
+calib_data = WaveformCalibrationContainer()
+status_data = PixelStatusContainer()
+
+channel = ['HG', 'LG']
+
+plot_dir = "none"
+
+
+def read_file(file_name, tel_id=1):
+    """
+     read camera calibration quantities
+
+     Parameters
+     ----------
+     file_name:   calibration hdf5 file
+
+     tel_id:      telescope id
+     """
+    with HDF5TableReader(file_name) as h5_table:
+        assert h5_table._h5file.isopen==True
+
+        table = f"/tel_{tel_id}/flatfield"
+        next(h5_table.read(table, ff_data))
+        table = f"/tel_{tel_id}/calibration"
+        next(h5_table.read(table, calib_data))
+        table = f"/tel_{tel_id}/pedestal"
+        next(h5_table.read(table, ped_data))
+        table = f"/tel_{tel_id}/pixel_status"
+        next(h5_table.read(table, status_data))
+
+    h5_table.close()
+
+
+def plot_all(ped_data, ff_data, calib_data, run=0, plot_file="none"):
```

Comment:
why not `plot_file=None` if you don't want to save the figure?

Reply:
I still think proper `None` would be better than a string but no big deal, as you prefer.